background.ts

```diff
@@ -17,6 +17,7 @@ import { getPathManagerFor, PathManagementStrategy, PathManager } from '@/integr
 import { CommandWorkerInterface, HttpCommandServer } from '@/main/commandServer/httpCommandServer';
 import SettingsValidator from '@/main/commandServer/settingsValidator';
 import { HttpCredentialHelperServer } from '@/main/credentialServer/httpCredentialHelperServer';
+import { init } from '@/main/dashboardServer';
```

Comment:
Please rename this to something clearer at the call site:
```suggestion
import { init as initDashboardServer } from '@/main/dashboardServer';
```

Reply:
I did quite a bit of refactoring to treat `DashboardServer` as a singleton, resulting in a major change in how we handle the import and init here.